libs/coin-evm/src/api/node/rpc.common.ts

```diff
@@ -249,7 +228,7 @@ export const getSubAccount: (
     const [balance, nonce, blockHeight] = await Promise.all([
       getCoinBalance(currency, addr),
       getTransactionCount(currency, addr),
-      api.getBlockNumber(),
+      api.getBlockNumber(), // @TODO GET RID OF THIS AND TAKE IT AS PARAM
```

Comment:
Might need to create a ticket for this, explain reasons etc

Reply:
Method has been remove as it's actually dead code. Probably a wrong copy paste from an old PR.